extension/chrome/elements/pgp_block.ts

```diff
@@ -235,7 +235,7 @@ Catch.try(async () => {
         await Store.dbContactSave(undefined, await Store.dbContactObj({ email: senderEmail, pubkey, client: pgpClient })); // <= TOFU auto-import
         render('Fetched pubkey, click to verify', () => window.location.reload());
       } else { // don't know who sent it
-        const { pubkey, pgpClient } = await Attester.lookupEmail(signerLongid);
+        const { pubkey, pgpClient } = await Keyserver.lookupLongid(acctEmail, signerLongid);
```

Comment:
The original `lookupEmail` was actually a typo, that still originally worked well because Attester accepts longids or email addresses interchangeably.